Vizzuality/half-earth-v3
# documentation: update sci-index and whitelisting code

## Added instructions for documentation and update whitelisting process
### Description
This PR only modifies the documentation for Scientists: 
1.  instructions to open the docs in local
2. instructions to add to the docs via prs
3. extended documentation for whitelisting services
### Testing instructions
Open the docs, check the scientist index and the whitelisting services. There is an image, the link shouldn't appear broken. 
### Feature relevant tickets
This documentation results from the task [HE-31](https://vizzuality.atlassian.net/browse/HE-31?atlOrigin=eyJpIjoiODljYzU1ZjBiODkzNGFhOWI3NTIyODYyYWM3YTY4ZjkiLCJwIjoiaiJ9)
